src/App.tsx

```diff
@@ -1,14 +1,74 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filters } from './helpers';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [filterBy, setFilterBy] = useState(Filters.All);
+  const [query, setQuery] = useState('');
+  const [error, setError] = useState('');
+
+  const handleSelectTodo = (todo: Todo) => {
+    setSelectedTodo(todo);
+  };
+
+  const filteredTodos = useMemo(
+    () => todos.filter(todo => {
+      const { completed, title } = todo;
+      const queryFilter = query === ''
+        || title.toLowerCase().includes(query);
+      let groupFilter;
```

Comment:
Prefer using `.toLowerCase()` on both `title` and `query` to make the comparison case insensitive and consistent 🦉.

Reply:
Fix Vadim's comment

Reply:
![image](https://user-images.githubusercontent.com/11507341/230398188-8dcb0cc0-3a89-4b06-8211-64bc74d8e2bb.png)
query is already formatted